src/style.css

```diff
@@ -1,3 +1,76 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  --global-color: #00acdc;
```

Comment:
Mind variable names - `global-color` is not informative name
Better choices are `active-color`, `active-link-color`, etc